### ISSUE TITLE
Some suggestions

### ISSUE THREAD

**Issue by ppoetsma** [bgeveritt/Notification#2]
Hi,

I have some suggestions for your widget to be able to use it as a kind of message box. Make the type configurable so when creating a new message you can specify the type. Add an xpath to enable use of this widget in multi-tenant / multi-user environments, or is the widget able to run within entity access? And add a run-this-microflow-when-clicked option to be able to 1) mark the notification as read and 2) run an additional microflow for additional processing.

Probably the domain model has to be extended to a Notification - Message pair where Notification is the dataview where the widget is in and it shows messages appearing in Message.

What I am looking for is a kind of message box where a user receives its notifications. Think of an envelope kind of image in the upper right corner and when click you see all active and past messages. While using the application, new messages popup using your widget to attract the user's attention. And when clicked the message is marked read and opened to view the details.

Is this what you have in mind with this widget? If you have questions let me know.

Regards,
Paul Poetsma
Gappless

**Comment by bgeveritt** [comment]
I agree it should be enhanced with an OnClick Microflow, but I think the functionality you'd be more interested in can be found here: http://codecanyon.net/item/noty-manager/7054200

**Comment by ppoetsma** [comment]
This is indeed what I am looking for. Thanks.
